### DISCUSSION TITLE
Timecard -> Order -> Invoice path may drop information like event dates

### DISCUSSION THREAD

**Discussion by sbts** [ledgersmb/LedgerSMB#5859] [Ideas]
When generating an order from a timecard, and an invoice from an order, some information seems to get lost.
In particular the date and staff member name don't make it to the invoice.

See [This Thread](https://sourceforge.net/p/ledger-smb/mailman/ledger-smb-users/thread/57A9DC7502000046000ED6A9%40mail.assuredcomp.com/#msg35271055) for some discussion started by Bob regarding this.

While we can (and should) release 1.5.0 without resolving this, I'd like to see it resolved before 1.5.1 if possible

**Comment by sbts** [comment]
![ledgersmb-sales-invoice](https://cloud.githubusercontent.com/assets/11002795/19459954/ba1615fc-9509-11e6-86c4-3a2b5dc10e98.png)
Is the mockup that Bob did of how he thinks date and staff name  etc should be shown on an invoice generated from a timecard

**Comment by ehuelsmann** [comment]
Great idea, but enhancements (as agreed recently) don't really belong on patch releases.
